css/ampdoc.css

```diff
@@ -392,3 +392,35 @@ amp-autocomplete > input,
     top: 200px;
   }
 }
+/**
+ * CSS for amp-addthis listing tool
+ */
+amp-addthis[data-widget-type="messages"] {
+  position: fixed !important;
+  width: 100% !important;
+  top: 0;
+}
+amp-addthis #closeButton {
+  right: 10px;
+  top: 10px;
+  width: 32px;
+  height: 32px;
+  opacity: 0.5;
+}
+amp-addthis #closeButton:hover {
+  opacity: 1;
+}
+amp-addthis #closeButton:before, #closeButton:after {
+  position: absolute;
+  content: ' ';
+  height: 20px;
+  width: 2px;
+  background-color: #fff;
+}
+amp-addthis #closeButton:before {
+  transform: rotate(45deg);
+}
+amp-addthis #closeButton:after {
+  transform: rotate(-45deg);
+}
```

Comment:
This should be in a new amp-addthis.css file in your extension, and then imported in amp-addthis.js.

```js
import {CSS} from '../../../build/amp-addthis-0.1.css';
```